src/routes/sessions/[id]/+page.svelte

```diff
@@ -177,6 +190,7 @@
 
 		<svelte:fragment slot="nav">
 			{#if !isNewSession}
+				<CopyButton content={sessionContent} title="Copy session" />
```

Comment:
I think we can simplify the implementation significantly by doing this:

```suggestion
				<CopyButton content={JSON.stringify(session.messages, null, 2)} />
```

Then we can get rid of the `copySessionContent()` function and the `sessionContent` variable.

The result will retain the structure of the session which is useful if you are pasting this session into new session as context or another LLM entirely:

```json
[
  {
    "role": "user",
    "content": "mic check, 1 2 3"
  },
  {
    "role": "ai",
    "content": "Checking the microphone input, 1...2...3."
  },
  {
    "role": "user",
    "content": "hello hello"
  },
  {
    "role": "ai",
    "content": "Hello! Is there anything I can assist you with today? If you have any questions or need assistance, please feel free to ask."
  }
]
```